relays/bin-substrate/src/cli/relay_headers_and_messages.rs

```diff
@@ -64,8 +63,6 @@ pub struct HeadersAndMessagesSharedParams {
 	/// Hex-encoded lane identifiers that should be served by the complex relay.
 	#[structopt(long, default_value = "00000000")]
 	lane: Vec<HexLaneId>,
-	#[structopt(long, possible_values = RelayerMode::VARIANTS, case_insensitive = true, default_value = "rational")]
```

Comment:
Please keep it - we have plans to use both modes. You may need to use runtime relayer configuration (vs compile-time) because of that - i.e.  instead of
```rust
pub type MessageLaneWococoMessagesToRococo = SubstrateMessageLaneToSubstrate<
	Wococo,
	WococoSigningParams,
	Rococo,
	RococoSigningParams,
	AltruisticStrategy,
>;
```
you'll need something like that:
```rust
MessageRelayer::new(AltruisticStrategy)
```

Reply:
I'll expand a bit on why we need altruistic strategy - it is for two cases: (1) serving special lanes, where paid fee is zero and (2) backup plan for fixing cases when rational relayers refuse to deliver messages and regular lanes are stuck (e.g. because of sudden conversion rate change).